packages/calcite-components/src/components/dialog/dialog.scss

```diff
@@ -78,27 +85,21 @@
 }
 
 :host([scale="s"]) {
-  --calcite-internal-dialog-content-padding: var(--calcite-spacing-sm);
-  --calcite-internal-dialog-padding: theme("spacing.3");
-  --calcite-internal-dialog-padding-large: theme("spacing.4");
-  --calcite-internal-dialog-title-text: theme("fontSize.1h");
-  --calcite-internal-dialog-content-text: theme("fontSize.n1");
+  --calcite-internal-dialog-content-padding: var(--calcite-dialog-content-space, var(--calcite-spacing-sm));
+  --calcite-internal-dialog-min-size-x: 198px;
+  --calcite-internal-dialog-min-size-y: 120px;
 }
 
 :host([scale="m"]) {
-  --calcite-internal-dialog-content-padding: var(--calcite-spacing-md);
-  --calcite-internal-dialog-padding: theme("spacing.4");
-  --calcite-internal-dialog-padding-large: theme("spacing.5");
-  --calcite-internal-dialog-title-text: theme("fontSize.2h");
-  --calcite-internal-dialog-content-text: theme("fontSize.0");
+  --calcite-internal-dialog-content-padding: var(--calcite-dialog-content-space, var(--calcite-spacing-md));
+  --calcite-internal-dialog-min-size-x: 288px;
+  --calcite-internal-dialog-min-size-y: 140px;
```

Comment:
cc @SkyeSeitz  @ashetland I think we may need to make these a bit taller - by default it allows shrinking too much with a common header / footer set up: 
![Screenshot 2024-08-26 at 4 30 04 PM](https://github.com/user-attachments/assets/9fd05d55-e282-419b-a735-426322306b89)

Reply:
Updated the [Figma specs ](https://www.figma.com/design/wVqftT1aZDjwYVkDZ2V5fC/Dialog?node-id=1423-25133&t=jeOtWMpcAdYzjKRY-11)with adjusted min-heights:
sm **140px**
md **180px**
lg **220px**
<img width="1171" alt="image" src="https://github.com/user-attachments/assets/7b8c6ef1-8fdc-4bfc-a9b4-121ca4634feb">